sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -1520,6 +1521,15 @@ operatorPipeRightSide
     | AGGREGATE namedExpressionSeq? aggregationClause?
     ;
 
+operatorPipeSetAssignmentSeq
+    : ident+=errorCapturingIdentifier
```

Comment:
`errorCapturingIdentifier` can be quoted identifiers. Shall we also test 
```
set `a.b` = ...
```
?

Reply:
Good idea, added a test case for this.